src/main/java/com/google/sps/servlets/LoadGameServlet.java

```diff
@@ -44,7 +44,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         String gameID = "demogameid";
 
         Game.Builder gameBuilder = new Game.Builder(gameID, "Demo");
-        gameBuilder.setGameCreator("username");
+        gameBuilder.setGameCreator("usernameid12345");
```

Comment:
For those hardcoded mock username, please consider to use constants and put all those constants at the beginning of this java file. In this way, it is easy to deprecate those mock fields in the future. Otherwise, you might need to go through the code to find those mock variables.

public class LoadGameServlet extends HttpServlet {
   private static final String USER_NAME = "usernameid12345";
   ....
         gameBuilder.setGameCreator(USER_NAME);
}

Reply:
Thanks, I will do this. I'm planning to move all this mock data into a mock data manager next, so I'll just set those variables there.